src/main/java/br/com/vivo/model/Conta.java

```diff
@@ -0,0 +1,45 @@
+package br.com.vivo.model;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "conta")
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
```

Comment:
coloca essa anotação -> @EqualsAndHashCode(of = "{id}")

Reply:
Feito!